packages/apps/client/src/api/ApiConnection.ts

```diff
@@ -40,10 +39,10 @@ export class APIConnection extends EventEmitter<APIConnectionEvents> {
 
 	private app: Application<AddTypeToProperties<ServiceTypes, SocketService>, unknown>
 
-	constructor(public readonly host = DEFAULT_DEV_API_HOST, public readonly port = DEFAULT_DEV_API_PORT) {
+	constructor() {

```

Comment:
I think we should probably make the API endpoint configurable, at least at the building stage? I think it would be smart  to build the capacity to mount this application anywhere from the start.

Reply:
I'm not opposed to that idea, but its a little tangential to the change made here.

The change made here was to stop connecting directly to the backend at `http://localhost:6500/` and instead to connect at `/` and rely on vite proxying to the backend during development.

This has no impact on whether it could be mounted in a subdirectory or not.  
Doing that requires some more thought/design/work. Presumably this should be set via an environment variable. How does that get into the ui?